react/features/recording/components/LiveStream/native/StreamKeyPicker.js

```diff
@@ -67,10 +67,22 @@ class StreamKeyPicker extends Component<Props, State> {
     render() {
         const { broadcasts } = this.props;
 
-        if (!broadcasts || !broadcasts.length) {
+        if (!broadcasts) {
             return null;
         }
 
+        if (!broadcasts.length) {
+            return (
+                <View style = { styles.formWrapper }>
+                    <View style = { styles.streamKeyPickerCta }>
```

Comment:
Is the double View tags necessary?

Reply:
Turns out they're not. I had copied the dom from below. I'll remove the extra view.